Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -271,6 +275,23 @@ public final class PostgresConnection {
         }
     }
 
+    static func makeBootstrap(
+        on eventLoop: EventLoop,
+        configuration: PostgresConnection.InternalConfiguration
+    ) -> NIOClientTCPBootstrapProtocol {
+        #if canImport(Network)
```

Comment:
I would prefer if clients were able to override this decision.

Reply:
Withdrawn, I didn't realize it was using the validating initializers. Good to go!